subprojects/test-runner/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/OrchestratorDelegate.kt

```diff
@@ -5,6 +5,13 @@ import androidx.test.internal.runner.RunnerArgsAccessor
 
 interface OrchestratorDelegate {
 
+    @Deprecated(
```

Comment:
@RuslanMingaliev Do you really need this change?

Reply:
don't think so